### ISSUE TITLE
ComboBox content outside window

### ISSUE THREAD

**Issue by redplane** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#1171]
Hi,

I'm using `MaterialDesignInXamlToolkit` to make a small app.
I have finished configuring material in my app and it worked.
But there is a trouble of `ComboBox`
When I make my `ComboBox` in `Grid` to be full-width, as long as I open the `ComboBox`, the context menu display out-side its container.

Here is howit looks:
![combobox](https://user-images.githubusercontent.com/7974095/51428021-9df05e00-1c39-11e9-8fc8-88ee43ed2d7a.png)

This is my `Login.xaml` in which I placed that `ComboBox`:

```
<Page x:Class="MainClient.View.Login"
    [..Declaration..]>

    <StackPanel>
        
        ...

        <Grid>
            <Grid.ColumnDefinitions>
                <ColumnDefinition Width="256" />
                <ColumnDefinition Width="46*" />
                <ColumnDefinition Width="51*" />
                <ColumnDefinition Width="175*" />
            </Grid.ColumnDefinitions>

            <TextBlock Text="{lex:Loc TITLE_LANGUAGE}" />
            <ComboBox Grid.Column="1"
                      ItemsSource="{Binding AvailableLanguages}"
                      SelectedItem="{Binding SelectedLanguage}" Grid.ColumnSpan="3">
                <ComboBox.ItemTemplate>
                    <DataTemplate>
                        <TextBlock Text="{Binding Path=Key, Converter={StaticResource LanguageTranslatorConverter}}" />
                    </DataTemplate>
                </ComboBox.ItemTemplate>
            </ComboBox>
        </Grid>

    </StackPanel>
</Page>
```

Here is my [REPO](https://bitbucket.org/Redplane/wpf-structure/src/master/)

Thanks

**Comment by stigrune** [comment]
This is how the ComboBox style in MaterialDesignInXamlToolkit is designed. It will use more width (and height) when the dropdown is open, then when closed.

My suggestion is to not use full-width for this, seems way too much space based on the content in your screenshot. If you really need it this wide, I would suggest adding a Margin on the ComboBox to keep the dropdown within the application frame.

**Comment by redplane** [comment]
Do you mean that we need to apply margin to the component that contains combobox ?

**Comment by Keboo** [comment]
@redplane there is also an attached property that you can set that will cause the popup with the items to show below combo box rather than around it.
```XAML
<ComboBox materialDesign:ComboBoxAssist.ClassicMode="True" />
```

**Comment by redplane** [comment]
> @redplane there is also an attached property that you can set that will cause the popup with the items to show below combo box rather than around it.
> 
> ```
> <ComboBox materialDesign:ComboBoxAssist.ClassicMode="True" />
> ```

Hmm...
I have tried your solution, but it is still displayed outside window.

![combobox](https://user-images.githubusercontent.com/7974095/51729698-bb954b80-20af-11e9-8cd9-e67c2f996c9c.png)


But it is Ok, when I set margin `Margin="0, 20, 0, 20", the `ComboBox` display is OK.
For me, it is acceptable, but the document should be updated about this :)

Thank you for your help.
